### PR TITLE
Update local dependencies

### PR DESCRIPTION
**Pull request by rohan-shettyy**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -49,6 +49,11 @@ if (hasProperty('cfLocal')) {
   def cfHome = String.valueOf(System.getenv('CHECKERFRAMEWORK'))
   dependencies {
     testImplementation files(cfHome + '/checker/dist/checker.jar')
+    testImplementation files(cfHome + '/checker/dist/checker-qual.jar')
+    testImplementation fileTree(dir: cfHome + '/framework-test/build/libs',
+                                include: '*.jar',
+                                exclude: ['*-sources.jar', '*-javadoc.jar'])
+    testImplementation libs.plume.utils
   }
```
Comment (wmdietl) [inline build.gradle:57]: Can you move the released dependencies to an else block here?
Currently, you first have the released dependencies and then add the local dependencies. In which order are these used?
It seems clearer to split these up cleanly.

File: build.gradle [hunk 2]
```diff
@@ -49,6 +49,11 @@ if (hasProperty('cfLocal')) {
   def cfHome = String.valueOf(System.getenv('CHECKERFRAMEWORK'))
   dependencies {
     testImplementation files(cfHome + '/checker/dist/checker.jar')
+    testImplementation files(cfHome + '/checker/dist/checker-qual.jar')
+    testImplementation fileTree(dir: cfHome + '/framework-test/build/libs',
```
Comment (wmdietl) [inline build.gradle:53]: Would it be easy to raise a warning if this matches more than one jar file?

File: build.gradle [hunk 3]
```diff
@@ -49,6 +49,11 @@ if (hasProperty('cfLocal')) {
   def cfHome = String.valueOf(System.getenv('CHECKERFRAMEWORK'))
```
Comment (wmdietl) [inline build.gradle:49]: It's too far away to comment on, so let me comment here:
````
  // This dependency is used by the application.
  implementation libs.guava
````

The project setup is a bit odd - there is no actual application, only the test.
If it's easy to do, can you change the whole project into just the conformance runner?
If that's difficult, what is the above `implementation` dependency for?

**Review by wmdietl — CHANGES_REQUESTED** [review]
Thanks!

File: build.gradle (hunk 3 above)
Comment (rohan-shettyy) [reply build.gradle:42]: This dependency was unnecessary.

**Review by wmdietl — APPROVED** [review]
Thanks!

**Pull request merged**